Makefile

```
@@ -63,12 +63,14 @@ fmt:
 	@find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
 	@find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/celestiaorg
 	@go mod tidy -compat=1.17
+	@markdownlint -fq -c .markdownlint.yaml .
```

```suggestion
	@markdownlint --fix --quiet --config .markdownlint.yaml .
```

the long command option is clearer in a file that other developers will read. The short command option is faster when typing command in your shell. 

https://changelog.com/posts/use-long-flags-when-scripting

---

Good point